### DISCUSSION TITLE
midi: wildfire integration

### DISCUSSION THREAD

**Discussion by legluondunet** [CorsixTH/CorsixTH#2521] [Research Room]
Hello,

do you think Wildfire could be a good option for Corsix-TH to play midi music on all OS platform?
--> multiplatform
--> does not need root rights on Linux (userland)

https://www.mindwerks.net/projects/wildmidi/
https://github.com/Mindwerks/wildmidi

**Comment by TheCycoONE** [comment]
It might be a fine option. We currently use sdl-mixer which supports fluidsynth or timidity on linux. I personally find fluidsynth satisfies my needs on that platform (though the environment variable to use it is a bit of an easter egg).
What problems are you having that wildfire could solve?
